compiler/parser/parser.peg

```diff
@@ -865,29 +824,19 @@ MergeOp
     }
 
 OverOp
-  = (OVER / UNNEST) _ exprs:Exprs locals:Locals? body:Lateral? {
+   = (OVER / UNNEST) _ exprs:Exprs locals:Locals? (_ "into"i _)? __ body:ScopeBody? {
```

Comment:
So the `into` keyword is not necessary to have a body for over? If that's the case can we just get rid of `into`?

Reply:
My problem is that with this you can do `over this into` which I don't think makes sense.